src/main/java/lesson8Person/Person.java

```diff
@@ -0,0 +1,71 @@
+package lesson8Person;
+
+import java.time.*;
+import java.util.Objects;
+
+public class Person {
+    private final String name;
+    private final String surname;
+
+    // if I don't initialize these, I can't use the 'LocalDate.of' method, any solutions to that?
+    private int yearOfBirth = 0;
+    private Month monthOfBirth = Month.January;
+    private int dayOfBirth = 0;
+
+    private final LocalDate birthDate;
+    private double height;
+    private double weight;
+    private Period age;
+
+    public Person(String name, String surname, int yearOfBirth, Month monthOfBirth, int dayOfBirth,
```

Comment:
Zamiast osobno przyjmować dzień, miesiąc oraz rok i wewnątrz konstruktora wywoływać metodę LocalDate.of, lepiej od razu przyjmować LocalDate. Będziemy mieli mniej argumentów oraz mniej obliczeń w konstruktorze.